demisto_sdk/commands/validate/validators/PA_validators/PA104_is_valid_modules.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import ClassVar, Iterable, List
+
+from demisto_sdk.commands.common.constants import MODULES
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class IsValidModulesValidator(BaseValidator[ContentTypes]):
+    error_code = "PA104"
+    description = "Validate that the modules field include only labels from the list of allowed labels."
+    error_message = f"Module field can include only label from the following options: {', '.join(MODULES)}."
+    related_field = "modules"
+    is_auto_fixable = True
+    fix_message = "Removed the following label from the modules field: {0}."
+    non_approved_modules_dict: ClassVar[dict] = {}
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(", ".join(non_approved_modules)),
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if (non_approved_modules := self.get_non_approved_modules(content_item))  # type: ignore[union-attr, arg-type]
```

Comment:
can we avoid those mypy errors in some way?

Reply:
I'll consult @ilaner